core/src/main/java/com/google/googlejavaformat/java/Replacement.java

```diff
@@ -23,28 +26,20 @@
  * <p>google-java-format doesn't depend on AutoValue, to allow AutoValue to depend on
  * google-java-format.
  */
-public class Replacement {
+public final class Replacement {
 
   public static Replacement create(int startPosition, int endPosition, String replaceWith) {
+    checkArgument(startPosition >= 0, "startPosition must be non-negative");
+    checkArgument(startPosition <= endPosition, "startPosition cannot be after endPosition");
     return new Replacement(Range.closedOpen(startPosition, endPosition), replaceWith);
   }
 
-  public static Replacement create(Range<Integer> range, String replaceWith) {
-    return new Replacement(range, replaceWith);
-  }
-
```

Comment:
Hi @awturner and @netdpb, please correct me if I've missed something, but it looks to me that the removal of `Replacement.create(Range, String)` and the introduction of the `final` keyword is an API-backwards-incompatible change. Thus I'm wondering if either it should be reverted or if the next version of google-java-format should be "2.0" to indicate through SemVer rules that a breaking change has occurred.

What do you think? :)